extensions/catalog/catalog-atlas/src/main/java/com/microsoft/dagx/catalog/atlas/metadata/AtlasDataCatalog.java

```diff
@@ -8,39 +8,18 @@
 import com.microsoft.dagx.spi.types.domain.metadata.DataCatalog;
 import com.microsoft.dagx.spi.types.domain.transfer.DataAddress;
 
-import java.util.Map;
-import java.util.stream.Collectors;
-
 public class AtlasDataCatalog implements DataCatalog {
 
-    private static final String ATLAS_PROPERTY_KEYNAME = "keyName";
-    private static final String ATLAS_PROPERTY_TYPE = "type";
-    private final AtlasApi atlasApi;
+    private final DataAddress address;
+
+    public AtlasDataCatalog(DataAddress address) {
 
-    public AtlasDataCatalog(AtlasApi atlasApi) {
-        this.atlasApi = atlasApi;
+        this.address = address;
     }
 
     @Override
     public DataAddress getPropertiesForEntity(String id) {
-        var entity = atlasApi.getEntityById(id);
-
-        if (entity != null) {
-            return DataAddress.Builder.newInstance()
-                    .keyName(entity.getEntity().getAttribute(ATLAS_PROPERTY_KEYNAME).toString())
-                    .type(entity.getEntity().getAttribute(ATLAS_PROPERTY_TYPE).toString())
-                    .properties(convert(entity.getEntity().getAttributes()))
-                    .build();
-        }
-
-        return null;
+        return address;
```

Comment:
Why return here the address?

Reply:
The `DataCatalog` is a bit of a misnomer, it is in fact the catalog_entry_. Therefore all inheritors should be renamed accordingly.